salt/kubernetes-master/addons/kubedns.yaml.jinja

```diff
@@ -0,0 +1,144 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kube-dns
+  namespace: kube-system
+  labels:
+    k8s-app: kube-dns
+    kubernetes.io/cluster-service: "true"
+spec:
+  replicas: {{ pillar['dns']['replicas'] }}
+  strategy:
+    rollingUpdate:
+      maxSurge: 10%
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      k8s-app: kube-dns
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-dns
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      volumes:
+      - name: kube-dns-config
+        configMap:
+          name: kube-dns
+      containers:
+      - name: kubedns
+        image: gcr.io/google_containers/k8s-dns-kube-dns-amd64:1.14.1
```

Comment:
Devel:CASP:1.0:ControllerNode is for 1.0, these are for 2.0, right?
 Devel:CASP:Head:ControllerNode would be right IBS project :)

Anyway - as long as they exist, no issues with landing it... It's only if they didn't as was going to be weeks or more I'd be concerned.

However, it is possible CI will start to timeout when the Provo internet connection is a little busy (which is often), so we should aim to land the images into the ISO and images ASAP.